test/testUtils.ts

```diff
@@ -13,9 +13,10 @@ const Providers = ({ children }) => {
   //   </ThemeProvider>
   // )
 };
-
-const customRender = (ui, options = {}) =>
+/* eslint-disable  @typescript-eslint/explicit-module-boundary-types */
+const customRender = (ui : JSX.Element, options = {}) =>
   render(ui, { wrapper: Providers, ...options });
+/* eslint-enable  @typescript-eslint/explicit-module-boundary-types */
```

Comment:
Just curious, why does this need to be here? I think some of this was boilerplate code.

Reply:
Yeah, the linter was yelling at the boilerplate code for not defining the type of the customRender function.

Reply:
If I run `yarn lint` in main, the linter fails in the current deployment because of this (and some extra things that were addressed in this PR as well).

Reply:
Oh I meant more - this seems wrong for boilerplate code, are we using it improperly? Something we can resolve outside of this PR!